Packs/Core/Playbooks/playbook-A_mail_forwarding_rule_was_configured_in_Google_Workspace.yml

```diff
@@ -0,0 +1,1783 @@
+      version: -1
+      name: Soft Response
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "10"
+      - "49"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 710,
+          "y": 1400
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "27":
+    id: "27"
+    taskid: 68992710-f44d-47dc-8ddb-82e8cea3339c
+    type: title
+    task:
+      id: 68992710-f44d-47dc-8ddb-82e8cea3339c
+      version: -1
+      name: Hard Response
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "29"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 160,
+          "y": 1400
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "29":
+    id: "29"
+    taskid: 10a29f38-0fc1-4df2-82bc-e7afb761788b
+    type: collection
+    task:
+      id: 10a29f38-0fc1-4df2-82bc-e7afb761788b
+      version: -1
+      name: Decide Whether to Suspend User Account
+      type: collection
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "33"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 160,
+          "y": 1530
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    message:
+      to:
+      subject:
+      body:
+      methods: []
+      format: ""
+      bcc:
+      cc:
+      timings:
+        retriescount: 2
+        retriesinterval: 360
+        completeafterreplies: 1
+        completeafterv2: true
+        completeaftersla: false
+    form:
+      questions:
+      - id: "0"
+        label: ""
+        labelarg:
+          simple: "The following evidences was found: \n\n${Evidences}\n\nWould you like to suspend the account ${Core.OriginalAlert.raw_abioc.event.identity_name} using Google Workspace Admin?"
+        required: false
+        gridcolumns: []
+        defaultrows: []
+        type: singleSelect
+        options: []
+        optionsarg:
+        - {}
+        - simple: "Yes"
+        - simple: 'No '
+        fieldassociated: ""
+        placeholder: ""
+        tooltip: ""
+        readonly: false
+      title: Select user account containment steps
+      description: The investigation revealed several suspicious indicators suggesting the user who created the forwarding rule may be compromised. The associated forwarding email and filters have been automatically removed. Please decide whether to take any additional recommended actions.
```

Comment:
The investigation identified several suspicious indicators, suggesting that the user who created the forwarding rule may have been compromised. The forwarding email and associated filters have been automatically removed. Please review and decide if any additional actions should be taken.